oci/runtime_oci.go

```diff
@@ -35,6 +35,9 @@ const (
 
 	// Command line flag used to specify the run root directory
 	rootFlag = "--root"
+
+	// default run root to be used for OCI runtime if it is not set by handler
+	defaultRunRoot = "/run/runc"
```

Comment:
Can or should this be in a conf file so it could be changed by the admin if necessary?